pajbot/config.py

```diff
@@ -1,13 +1,17 @@
 from typing import Dict, Tuple, Type
 
 from pajbot.managers.twitter import GenericTwitterManager, PBTwitterManager, TwitterManager
+from pajbot.apiwrappers.twitch.helix import TwitchHelixAPI
 
 ConfigSection = Dict[str, str]
 Config = Dict[str, ConfigSection]
 
 
 def load_streamer_and_channel(config: Config) -> Tuple[str, str]:
-    if "streamer" in config["main"]:
+    if "streamer_id" in config["main"]:
+        streamer = TwitchHelixAPI.get_login(config["main"]["streamer_id"])
```

Comment:
No API requests should take place in this file

Reply:
Also, this is not going to work, TwitchHelixAPI.get_login needs a `self` parameter so it needs to be called on an instance of the API